sdk/include/ndk/rtltypes.h

```diff
@@ -1423,7 +1423,17 @@ typedef struct _RTL_CRITICAL_SECTION_DEBUG
     LIST_ENTRY ProcessLocksList;
     ULONG EntryCount;
     ULONG ContentionCount;
-    ULONG Spare[2];
+    union
+    {
+        ULONG_PTR WineDebugString;
```

Comment:
Do we need the Wine-specific stuff in the NDK ? Normally, Wine code doesn't (or shouldn't....) use the NDK headers? Or do we have code that does that (or is not wine but where some parts use a Wine-style CRITICAL section debugging)?

Reply:
We will have some code that accesses this, and accessing `WineDebugString` is infinitely more readable than `Spare[0]` which wine does do a lot.